lib/faker.ex

```diff
@@ -76,14 +76,100 @@ defmodule Faker do
     Application.put_env(:faker, :locale, lang)
   end
 
+  @doc """
+  Returns a positive integer between 0 and total.
+
+  ## Examples
+
+      iex> random(5) in [0, 1, 2, 3, 4, 5]
+      true
+  """
+  @spec random(integer) :: integer
+  def random(total) do
```

Comment:
While your example does hint that the random function should return an integer between 0 and `total` (inclusive), it is not clear it does. I'd suggest mentioning it explicitly in the doc:

> Returns an integer between 0 and `n`

Additionally, I feel `n` to be a better variable name than `total`, but it doesn't really matter to me